docs/resources/digital_wallet_application.md

```diff
@@ -56,10 +56,13 @@ resource "pingone_digital_wallet_application" "my_digital_wallet_app" {
 ### Required
 
 - `app_open_url` (String) The URL enables deep-linking to the digital wallet application, and is sent in notifications to the user to communicate with the service.
-- `application_id` (String) The identifier (UUID) of the PingOne mobile application associated with the digital wallet application.
```

Comment:
Just to check this should be an optional attribute now?

Reply:
Prior change was incorrect. The `application_id` is now properly defined as required.